pkg/hintrunner/zero/zerohint_dictionaries_test.go

```diff
@@ -0,0 +1,39 @@
+package zero
+
+import (
+	"testing"
+
+	"github.com/NethermindEth/cairo-vm-go/pkg/hintrunner/hinter"
+	"github.com/consensys/gnark-crypto/ecc/stark-curve/fp"
+)
+
+func TestZeroHintDictionaries(t *testing.T) {
+	runHinterTests(t, map[string][]hintTestCase{
+		"SquashDictInnerAssertKeys": {
+			{
+				operanders: []*hintOperander{},
+				ctxInit: func(ctx *hinter.HintRunnerContext) {
+					ctx.ScopeManager.EnterScope(map[string]any{"keys": []fp.Element{}})
+					err := hinter.InitializeSquashedDictionaryManager(ctx)
```

Comment:
Why is it needed to call `InitializeSquashedDictionaryManager` here? 
If we purely test the hint, then we just need to enter the scope with keys.

If you remove it , test pass. `InitializeSquashedDictionaryManager`   deals with the `SquashedDictionaryManager` which shouldn't be managed manually for this hint anyway as previously discussed.

You dont use it in the second test, i guess it can be removed ? @har777  @cicr99

Reply:
Yep just `hinter.InitializeScopeManager(ctx, map[string]any{"keys": []fp.Element{}})` is enough

Reply:
thank you, will fix it!

Reply:
sure, you are right!
`InitializeSquashedDictionaryManager` is beyond the hint implementation
I don't need it to test the hint because the hint itself is a granular and independent part from the DictionaryManager and I only need a scope to test the hint.
Is that correct?

Reply:
@TAdev0 @har777 new changes were pushed! 
thank you, I value a lot your feedback!